src/Views/CheckView.vala

```diff
@@ -99,6 +105,18 @@ public class Installer.CheckView : AbstractInstallerView {
             minimum_specs = false;
         }
 
+        var apt_sources = File.new_for_path ("/etc/apt/sources.list.d/elementary.list");
```

Comment:
In onboarding we read is info: https://github.com/elementary/onboarding/blob/master/src/MainWindow.vala#L53

Did you already try this here? Would be nice if we could use the same logic.

Reply:
Yeah, this seems flawed though because lsb info comes from the os-patches repo which is applied to both daily and stable, so this stops working as soon as we set LSB to not say "early access"